cnxarchive/views.py

```diff
@@ -981,3 +1008,46 @@ def robots(request):
     resp.expires = html_date(exp_time)
     resp.body = robots_dot_txt.to_string()
     return resp
+
+
+@view_config(route_name='recent', request_method='GET',
+             accept="application/json",
+             renderer='templates/recent.rss')
+def recent(request):
+    # setting the query variables
+    num_entries = request.GET.get('number', 10)
+    start_entry = request.GET.get('start', 0)
+    portal_type = request.GET.get('type', ['Collection', 'Module'])
+    if portal_type != ['Collection', 'Module']:
+        portal_type = [portal_type]
+    # search the database
+    settings = request.registry.settings
+    statement = """
+                SELECT name, revised, authors, abstract,
+                'http://cnx.org/contents/'||
+                ident_hash( uuid, major_version, minor_version) AS link
```

Comment:
changed to ignore major and minor version because just uuid gives the most recent version anyway. So outside of the sql it does http://cnx.org/contents/ + uuid